src/sql/parts/notebook/notebookStyles.ts

```diff
@@ -99,7 +99,7 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean): IDi
 			collector.addRule(`
 				.notebookEditor .notebook-cell {
 					border-color: ${inactiveBorder};
-					border-width: 0px;
+					border-width: 1px;
```

Comment:
Ah cool, so we had different width in the 2 cases? Now it's always consistently 1px? or can you explain how this works so I understand CSS better? Thanks 😄

Reply:
Now it’s consistently 1px (instead of alternating between 0 and 1).

I used this as inspiration: https://stackoverflow.com/questions/9612758/add-a-css-border-on-hover-without-moving-the-element

Reply:
Awesome. :shipit: